lib/model.jsonschema.js

```diff
@@ -73,6 +73,16 @@ const compatibilityModelJsonSchema = {
       default: STATES.DRAFT,
       description: 'define which is the value assigned to field __STATE__ of newly created records whenever such field is not provided',
     },
+    defaultSorting: {
+      type: 'object',
+      description: 'default sorting applied to CRUD collection',
+      patternProperties: {
+        '^.*$': {
+          type: 'number',
+          enum: [-1, 1],
+        },
+      },
+    },
```

Comment:
```suggestion
    defaultSorting: {
      type: 'object',
      description: 'default sorting applied to CRUD collection',
      additionalProperties: {
        type: 'number',
        enum: [-1, 1],
      },
    },
```
Since we're not interest in enforcing a specific pattern on the property name, but rather constrain the object schema, we may employ the [`additionalProperties`](https://json-schema.org/understanding-json-schema/reference/object#additionalproperties) insteadOf `patternProperties`. What do you think?

Reply:
Looks much clearer, thx!